<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTaskDialog</class>
 <widget class="QDialog" name="NewTaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建任务</string>
  </property>
  <widget class="QGroupBox" name="gBTaskinfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>259</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lebelStore">
        <property name="maximumSize">
         <size>
          <width>116</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>零售店：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combStore">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>服务类型:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combServType">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEUserName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEPasswd">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>身份证号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEGovId"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>手机号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEPhoneNumber"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>预约码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEReservCode"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="pBOk">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBImport">
        <property name="text">
         <string>导入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBClear">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBOk</sender>
   <signal>clicked()</signal>
   <receiver>NewTaskDialog</receiver>
   <slot>startTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBImport</sender>
   <signal>clicked()</signal>
   <receiver>NewTaskDialog</receiver>
   <slot>importUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startTask()</slot>
  <slot>importUser()</slot>
 </slots>
</ui>
